utils/time.py

```diff
@@ -1,11 +1,12 @@
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
 
+time_format = "%m/%d/%Y %H:%M:%S"
 
 class Time:
```

Comment:
This class is only used by tests, hence the reason for it being in the test folder. I don't think it's appropriate to move it into the application code. This change will not block merging though.

Reply:
Having all the time components in one place helps with the organization I think. Plus we can use that class in other places like the Widgets functions when we need to

Reply:
Then we should probably at some point create unit tests for the class.